hw_5.py

```diff
@@ -0,0 +1,98 @@
+# 1. Из текстового файла удалить все слова, содержащие от трех до пяти символов,
+# но при этом из каждой строки должно быть удалено только четное количество таких слов.
+
+def word_in_file_deleter(f_name):
```

Comment:
Also need to think about punctuations in this file, because if I add a comma or dot, then this word not will be deleted